api/oneclick.py

```diff
@@ -800,8 +802,9 @@ def process_availability(self, media_type='ebook'):
             license_pool, is_new, is_changed = self.api.update_licensepool_for_identifier(isbn, available)
             # Log a circulation event for this work.
             if is_new:
-                Analytics.collect_event(
-                    self._db, license_pool, CirculationEvent.DISTRIBUTOR_AVAILABILITY_NOTIFY, license_pool.last_checked)
+                for library in self.collection.libraries:
+                    self.analytics.collect_event(
+                        library, license_pool, CirculationEvent.DISTRIBUTOR_TITLE_ADD, license_pool.last_checked)
```

Comment:
Yes, this should be TITLE_ADD. Sounds like someone was thinking the "availability" meant availability _in general_.